src/neptune/internal/utils/dependency_tracking.py

```diff
@@ -4,10 +4,10 @@
     "FileDependenciesStrategy",
 ]
 
+import logging
```

Comment:
We have our custom logger somewhere. It has some settings to simplify filtering etc. I think it's:
```python
from neptune.internal.utils.logger import logger
```

Reply:
I tried that at first and it doesn't show the level - just the message. I could change the logging format, but that would affect other usages of this logger throughout the code base, right?